tests/windows_test.go

```diff
@@ -62,7 +62,7 @@ var _ = Describe("[Serial][sig-compute]Windows VirtualMachineInstance", func() {
 	var virtClient kubecli.KubevirtClient
 	var windowsVMI *v1.VirtualMachineInstance
 
-	BeforeEach(func() {
+	BeforeEach(OncePerOrdered, func() {
```

Comment:
`BeforeEach` on line 82 can be also be `OncePerOrdered` ?

Reply:
+1